de.dlr.sc.virsat.external.lib.json-simple/src/de/dlr/virsat/external/lib/json/simple/Activator.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.virsat.external.lib.json.simple;
+
+import de.dlr.sc.virsat.external.lib.LibPlugin;
+
+/**
+ * The plugin root class which is the entry point for the OSGi bundle activation
+ * @author fisc_ph
+ *
+ */
+public class Activator extends LibPlugin {

```

Comment:
Does this Activator need to be derived from LibPlugin? Isn't this only needed for Libs which access java external bianries such as z3 or vtk does? Please corsscheck with an external lib such as freechart or any other etc.

Reply:
We checked JFreeChart and Spotbugs which also only have hava binaries and decided to keep it the same here for consistency with them.